packages/wallet/src/keystore-wallet.ts

```diff
@@ -90,7 +89,7 @@ export async function encryptKeystoreWallet(
 
   // Construct keystore.
   const keystore: KeystoreWallet = {
-    id: uuidv4(),
+    id: crypto.randomUUID(),
```

Comment:
@nedsalk Did you test it on a browser? It might be ok as it is, but we must be sure. Before #1431 goes in, I'm afraid browser tests must be done manually.

@Torres-ssf Would it make sense to export this method from the [`crypto`](https://github.com/FuelLabs/fuels-ts/tree/master/packages/crypto/src) package?

Reply:
https://github.com/FuelLabs/fuels-ts/pull/1608/commits/2cb31dc4fc4956174a840ccde45fd4bdc08f44d6

I exported it from `crypto` in the commit above. I tried out `crypto.randomUUID()` in my console and it worked. Don't see why it wouldn't work the same way the other functions that depend on the browser's `crypto` API work.

Reply:
A thing to note is that the CI [failed](https://github.com/FuelLabs/fuels-ts/actions/runs/7410773675/job/20163803909#step:9:449) on node v18, but passed on node v20. Perhaps the code can be simplified when we drop support for v18 in the far future.

Reply:
Not sure if I am tripping but taking a look at the [Browser compatibility](https://developer.mozilla.org/en-US/docs/Web/API/Crypto/randomUUID#browser_compatibility) section for `ramdomUUID` I've just noticed that the Crypto API and all its methods/helpers are all supported in different versions among different browser.

I am raising this concern because I am not sure if we need to improve the browser validations [here](https://github.com/FuelLabs/fuels-ts/blob/master/packages/crypto/src/browser/crypto.ts) to ensure that `subtle` is present on Crypto and, from now on, `randomUUID`

Reply:
Hmmm, the browser support for `randomUUID` started from July 2021 to March 2022, depending on the browser, whereas the other `crypto` utilities have been available for quite some time. I'm not sure if we should go this route anymore.

@arboleya, you've been developing for the interwebs for ages :D, what do you think? Is ~2 years of existing support acceptable? If it's raising alarms for you, let's drop the PR.

Reply:
@Torres-ssf Good point. It's safe to validate everything in the [crypto.ts](https://github.com/FuelLabs/fuels-ts/blob/master/packages/crypto/src/browser/crypto.ts) file—`subtle`, `randomUUID`, etc.

@nedsalk By addressing Sergio's concerns, I believe we'll be good to go.

Reply:
https://github.com/FuelLabs/fuels-ts/pull/1608/commits/6f6b856a22581e2e3acb6c6a84555fa66c74d6c6